LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabClient.java

```diff
@@ -149,6 +150,28 @@ private String valueOf(final Object obj) {
 
     @Override
     public ILabAccount createTempAccount(@NonNull final TempUserType tempUserType) throws LabApiException {
+        // Adding a second attempt here, api sometimes fails to create the temp user.
+        try {
+            return createTempAccountInternal(tempUserType);
+        } catch (LabApiException e){
+            if (e.getErrorCode().equals(LabError.FAILED_TO_CREATE_TEMP_USER)){
+
+                // Wait for a bit
+                try {
```

Comment:
The failure in creating the temp user could be tied to a network issue, the wait is to allow the device some time to reconnect before retrying. I also found that this would increase reliability for the most part, I don't see this error anymore.